tests/e2e/config/performance-reporter.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+const { readFileSync, statSync } = require( 'fs' );
+const chalk = require( 'chalk' );
+const { PERFORMANCE_REPORT_FILENAME } = require( '../../utils/constants' );
+
+class PerformanceReporter {
+	onRunComplete() {
+		if ( statSync( PERFORMANCE_REPORT_FILENAME ).size === 0 ) {
+			return;
+		}
+		const reportFileContents = readFileSync( PERFORMANCE_REPORT_FILENAME )
+			.toString()
+			.split( '\n' )
+			.slice( 0, -1 )
+			.map( ( line ) => JSON.parse( line ) );
+
+		reportFileContents.forEach( ( testReport ) => {
+			// eslint-disable-next-line no-console
+			console.log(
+				chalk.bgGreenBright.underline.bold( testReport.description )
```

Comment:
The contrast of light green with white text is pretty hard to read. Maybe we can find a darker green?